v4/source/channels.yaml

```diff
@@ -1330,3 +1330,56 @@
 
             // GetChannelUnread
             channelUnread, resp := Client.GetChannelUnread(<CHANNELID>, <USERID>)
+
+'/channels/{channel_id}/schemes/{scheme_id}':
+    put:
+      tags:
+        - channels
+      summary: Set a channel's scheme
+      description: |
+        Set a channel's scheme.
```

Comment:
I would like to have a more clear explanation here. Without context is difficult to understand that you are setting the schema_id as schema of the channel.